### DISCUSSION TITLE
Add telegram and whatsapp share icon

### DISCUSSION THREAD

**Discussion by jundi14** [nunocoracao/blowfish#643] [Ideas]
Hello

I wanted to start off by saying that this theme is great! thanks @nunocoracao 

I'm from Indonesia, in my country a lot of people use whatsapp and also telegram.
I think adding the share icon for both of them can make this theme extraordinarily good.

put this code on `data/sharing.json`

```
"whatsapp": {
    "icon": "whatsapp",
    "title": "sharing.whatsapp",
    "url": "https://api.whatsapp.com/send?text=%s&resubmit=true&title=%s"
  }
```
and
 ```
"telegram": {
    "icon": "telegram",
    "title": "sharing.telegram",
    "url": "https://t.me/share/url?url=%s&resubmit=true&title=%s"
  }
```
and add 

```
"whatsapp"
``` 
and 
```
"telegram"
```
in `article.Sharinglinks`

for example
`sharingLinks = ["twitter", "pinterest", "facebook", "email", "whatsapp", "telegram"]`

on `params.toml`

sorry if my language is not good, this is also my first time doing PR. Thank You
***
Result

![Screenshot (250)](https://user-images.githubusercontent.com/57140394/228713116-62466f31-2bc0-44dd-8596-c10be1f73c7c.png)

**Comment by nunocoracao** [comment]
Thanks @jundi14 created https://github.com/nunocoracao/blowfish/issues/646 to track this change
